src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetSemanticInfoTests.cs

```diff
@@ -268,21 +267,48 @@ class K
 ";
             var semanticInfo = GetSemanticInfoForTest(sourceCode);
 
-            Assert.Equal("System.Int32", semanticInfo.Type.ToTestDisplayString());
-            Assert.Equal(TypeKind.Struct, semanticInfo.Type.TypeKind);
-            Assert.Equal("System.Int32", semanticInfo.ConvertedType.ToTestDisplayString());
-            Assert.Equal(TypeKind.Struct, semanticInfo.ConvertedType.TypeKind);
+            Assert.Null(semanticInfo.Type);
+            Assert.Null(semanticInfo.ConvertedType);
             Assert.Equal(ConversionKind.Identity, semanticInfo.ImplicitConversion.Kind);
 
             Assert.Null(semanticInfo.Symbol);
-            Assert.Equal(CandidateReason.NotInvocable, semanticInfo.CandidateReason);
-            Assert.Equal(1, semanticInfo.CandidateSymbols.Length);
-            var sortedCandidates = semanticInfo.CandidateSymbols.OrderBy(s => s.ToTestDisplayString(), StringComparer.Ordinal).ToArray();
-            Assert.Equal("System.Int32 K.f", sortedCandidates[0].ToTestDisplayString());
-            Assert.Equal(SymbolKind.Field, sortedCandidates[0].Kind);
+            Assert.Equal(CandidateReason.None, semanticInfo.CandidateReason);
+            Assert.Empty(semanticInfo.CandidateSymbols);
```

Comment:
>Assert.Empty(semanticInfo.CandidateSymbols); [](http://example.com/codeflow?start=12&length=44)

It looks like new behavior is worse.

Reply:
Yes, slightly. Note that's the behavior we have already for instance receiver (the test I added below passes as-is before this PR). I'll add a PROTOTYPE comment to explore tweaking that if possible.